index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="/src/assets/style.css" />
+		<link rel="stylesheet" href="/index.css" />
+		<script src="https://miro.com/app/static/sdk/v2/miro.js"></script>
+		<title>CB-Miro v0.13.0</title>
+	</head>
+	<body>
+		<div class="grid container">
+			<div class="cs1 ce3">
+				<img
+					class="brand-logo.lg"
+					src="src/assets/images/codeBeamer-Logo.png"
+				/>
+			</div>
+
+			<div class="cs10 ce12">
+				<img
+					class="brand-logo-lg"
+					src="src/assets/images/miro-Logo.png"
+				/>
+			</div>
+			<div class="cs1 ce12">
+				<h1>
+					CodeBeamer / Miro Integration Plugin v0.13.0<br /><small
+						class="muted"
+						>by
+						<a href="https://github.com/max-poprawe">max-poprawe</a
```

Comment:
By now you can drop Max and put your name at the front as you are doing most of the work 😉

Reply:
Admittably, I had already sneakily already changed it to my name in `package.json`.  

Implemented as suggested, pushed right to current working branch `refactor/raect/mvp` to not have to deal with additional rebasing/merging.